cmd/iam.go

```diff
@@ -1475,6 +1477,87 @@ func (sys *IAMSys) updateGroupMembershipsForLDAP(ctx context.Context) {
 	}
 }
 
+// ValidateLDAPMappingImport - validates the LDAP policy mappings. Keys in the
+// given map may not correspond to LDAP DNs - these keys are ignored.
+//
+// For validated mappings, it updates the key in the given map to be in
+// normalized form.
+func (sys *IAMSys) ValidateLDAPMappingImport(ctx context.Context, isGroup bool,
```

Comment:
s/ValidateLDAPMappingImport/NormalizeLDAPMappingImport

Validate can be confusing